src/controlles/productController.js

```diff
@@ -0,0 +1,23 @@
+const productService = require('../services/productService');
+const bcrypt = require("../middlewares/bcrypt");
+const verfiyToken = require("../middlewares/verfiyToken");
+
+//메인
+
+
+const selectProduct = async (req, res) => {
+    try {
+            const customerIf = req.user
+            //제품 목록 가져오기
+            const mainList = await productService.selectProduct(customerIf);
+            //데이터를 찾으면 상태 코드와 제품 목록을 json형식으로 클라이언트에 응답
+            return res.status(200).json(mainList);
+    } catch (err) {//에러 캐치하기
```

Comment:
```suggestion
            const mainList = await productService.selectProduct(customerIf);
            //데이터를 찾으면 상태 코드와 제품 목록을 json형식으로 클라이언트에 응답
            return res.status(200).json(mainList);
    } catch (err) {//에러 캐치하기
```
- mainList json객체로 보내실 때는 배열 통째로 반환하시면 안되고, key-value 형태를 맞춰서 보내주시는 게 좋습니다 :)
- mainList: ____ 라는 key를 갖도록 해주세요